packages/components/src/radio/src/Radio.css

```diff
@@ -49,80 +50,94 @@
 /* CONTENT | LABEL | REVERSE */
 .o-ui-radio-reverse .o-ui-radio-label {
     margin-left: 0;
-    margin-right: var(--o-ui-sp-2);
+    margin-right: var(--hop-space-inline-sm);
 }
 
 /* CONTENT | ICON */
 .o-ui-radio-icon {
-    margin-left: var(--o-ui-sp-1);
+    margin-left: var(--hop-space-inline-sm);
 }
 
 /* CONTENT | ICON | REVERSE */
 .o-ui-radio-reverse .o-ui-radio-icon {
     margin-left: 0;
-    margin-right: var(--o-ui-sp-1);
+    margin-right: var(--hop-space-inline-sm);
 }
 
 /* CONTENT | COUNTER */
 .o-ui-radio-counter {
-    margin-left: var(--o-ui-sp-2);
+    margin-left: var(--hop-space-inline-sm);
     color: var(--o-ui-text-alias-primary);
 }
 
 /* CONTENT | COUNTER | REVERSE */
 .o-ui-radio-reverse .o-ui-radio-counter {
     margin-left: 0;
-    margin-right: var(--o-ui-sp-2);
+    margin-right: var(--hop-neutral-text);
```

Comment:
This is not the right token.

Reply:
I don't see a use case for this in Figma what is the right token?

Reply:
😵